org.mypaint.MyPaint.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "org.mypaint.MyPaint",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
```

Comment:
I did it mainly to differentiate from the nightly manifest. But I will change it to 3.24, builds fine in both.